tests/codes/eurocode/nen_en_1993_5_2008/chapter_5_ultimate_limit_states/test_formula_5_2.py

```diff
@@ -0,0 +1,54 @@
+"""Testing formula 5.2 of NEN-EN 1993-5:2008."""
+
+import pytest
+
+from blueprints.codes.eurocode.nen_en_1993_5_2008.chapter_5_ultimate_limit_states.formula_5_2 import Form5Dot2DesignMomentResistanceClass1Or2
+from blueprints.validations import LessOrEqualToZeroError
+
+
+class TestForm5Dot2DesignMomentResistanceClass1Or2:
+    """Validation for formula 5.2 from NEN-EN 1993-5:2008."""
+
+    def test_evaluation(self) -> None:
+        """Test the evaluation of the result."""
+        # Example values
+        beta_b = 0.5  # Dimensionless
+        w_pl = 20  # MM3
+        f_y = 200  # MPA
+        gamma_m_0 = 0.8  # Dimensionless
+        form = Form5Dot2DesignMomentResistanceClass1Or2(beta_b=beta_b, w_pl=w_pl, f_y=f_y, gamma_m_0=gamma_m_0)
+
+        # Expected result, manually calculated
+        expected = 2.5
```

Comment:
```suggestion
        expected = 0.0025  # KNM
```
Or consider using a large w_pl if you don't want to get small calculated values